nuancier/lib/model.py

```diff
@@ -153,10 +153,10 @@ def api_repr(self, version):
                 id=self.id,
                 name=self.election_name,
                 year=self.election_year,
-                date_start=self.election_date_start,
-                date_end=self.election_date_end,
-                submission_date_start=self.submission_date_start,
-                submission_date_end=self.submission_date_end,
+                date_start=self.election_date_start.strftime('%Y-%m-%d %H:%M:%S') ,
```

Comment:
You could also use: ``.isoformat()``

Reply:
no isoformat?

Reply:
Good hint. Actually format "%Y-%m-%d %H:%M:%S" is wrong here, all fields are `date`, not `datetime`.